### ISSUE TITLE
Change to more permissive MIT license

### ISSUE THREAD

**Issue by emirkmo** [jm33-m0/win10-mouse-natural-scroll#2]
https://choosealicense.com/licenses/mit/#

Changing to MIT license would allow this to be used by people like Microsoft, who cannot use GPL licenses since it would force them to open source their code. 

While I choose GPL when I can, it would be nice if this was in PowerToys..

(see https://github.com/microsoft/PowerToys/issues/6434#issuecomment-1038224087)

**Comment by jm33-m0** [comment]
Makes sense, I just asked if they want to use my script, will change license if necessary
